addons/spreadsheet/static/tests/global_filters/global_filters_model_test.js

```diff
@@ -195,6 +202,59 @@ QUnit.module("spreadsheet > Global filters model", {}, () => {
         assert.deepEqual(result, DispatchResult.Success);
     });
 
+    QUnit.test(
```

Comment:
I also moved the tests from components test to model only tests. It's much much faster. (and I don't know why, the global filters test from the UI are particularly slow)